src/jumpstart/JumpstartEnterAmount.tsx

```diff
@@ -170,11 +172,13 @@ function JumpstartEnterAmount() {
       {sendAmountExceedsThreshold && (
         <InLineNotification
           variant={NotificationVariant.Warning}
-          title={t('jumpstartEnterAmountScreen.maxAmountWarning.title')}
-          description={t('jumpstartEnterAmountScreen.maxAmountWarning.description', {
-            maxAmount: maxSendAmountLocalCurrency?.toFormat(2),
-            localCurrencyCode,
+          title={t('jumpstartEnterAmountScreen.maxAmountWarning.title', {
+            amountInLocalCurrency: maxSendAmountLocalCurrency,
+            formatParams: {
+              amountInLocalCurrency: { currency: localCurrencyCode, locale },
+            },
```

Comment:
oh interesting, is `formatParams` a built-in i18next feature?

Reply:
ah yes https://www.i18next.com/translation-function/formatting
And do we have the necessary `Intl` support working in both iOS and Android RN env?

Reply:
i'll check android - on iOS it looks like it's working correctly!

Reply:
android is also working correctly - checked with currency set to euros, if the app language is en-US, then the currency format displayed is like "€100". if the language is de-DE, the format is "100 €"